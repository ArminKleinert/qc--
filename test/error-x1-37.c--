export bits32 z;

const {
    z = 3;
}
