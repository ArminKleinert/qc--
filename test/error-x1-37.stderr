file error-x1-37.c--, line 1, characters 1-17 Error: z can't be exported
